sql/07-query-friends-gabriel.sql

```diff
@@ -0,0 +1,13 @@
+SELECT 	
+	name 
+FROM 	
+	student.student
+JOIN 	
+	student.friend 
+ON 		
+	student.student.id = student.friend.id1
+WHERE 	
+	id2 = 1911 
```

Comment:
The spirit of this task is to query everyone who is friends with someone named Gabriel, so you would probably want the where clause to be on the name.  Hardcoding it for the two student IDs of people named Gabriel is manual (so it's actually more work), and your query doesn't reflect the spirit of the question, and it will break if there are any new people named Gabriel.  Ideally, someone would be able to tell what you're trying to do by reading your query.